static/es/index.template.html

```diff
@@ -194,26 +194,33 @@ <h3>🌡️ Sleuth-o-meter</h3>
 			</div>
 
 			<p>Cada pista revelada disminuirá tu puntaje. Ten en cuenta que el sitio web aún está en beta, por lo que este sistema de puntuación, la interfaz o el diseño están sujetos a cambios.
-				Siéntete libre de proporcionar comentarios <a class="link-dark" href="https://forms.gle/dvA4Wr8LiuHFmDZN7">aquí</a> o en <a
+				No dudes en enviarnos tus comentarios <a class="link-dark" href="https://forms.gle/dvA4Wr8LiuHFmDZN7">aquí</a> o en <a
 				class="link-dark" href="https://www.reddit.com/r/mystery_o_matic/">nuestro subreddit</a>.
 			</p>
 		</div>
 	</div>
 
 
-	<div id="how-to-play" style="display:none">
+	<div id="how-to-play">
 		<div class="container mt-5">
-			<p>mystery-o-matic presenta un nuevo misterio cada día, ambientado en un lugar específico, donde cada sospechoso
-				actúa de forma independiente. Las pistas proporcionadas requieren deducciones espaciales
-				espaciales y temporales para resolver el caso.</p>
+			<p class="fst-italic"> La pandilla se reúne alrededor de un tomo polvoriento adornado con extravagantes letras doradas en la portada que dicen <b>"Mystery-o-Matic"</b>.</p>
+			<p class="fst-italic">
+				—¿Por qué quieren que leamos esto? —pregunta Alice mientras sopla el polvo de la tapa.
```

Comment:
Use an html entity instead of unicode symbol (`—`)

Reply:
[Actually..](https://stackoverflow.com/questions/9808098/html5-which-is-better-using-a-character-entity-vs-using-a-character-directly) :thinking:

Reply:
Ok, cuando termine todas las traducciones puedo pasarle el texto a ChatGPT a ver si me lo deja con guión o si me lo cambia por comillas, ya que es una regla que siempre confundo en español.